.storybook/main.js

```diff
@@ -6,8 +6,11 @@ const AppSourceDir = path.join(__dirname, '..', 'src');
 module.exports = {
     stories:
         process.env.NODE_ENV === 'production'
-            ? ['../src/**/*.stories.tsx']
-            : ['../src/**/*.stories.tsx', '../src/**/*.stories.local.tsx'],
+            ? ['../src/Apps/**/*.stories.tsx']
+            : [
```

Comment:
won't this hide everything in storybook except Legion?

Reply:
to reduce build times for this feature branch